data/json/mutations.json

```diff
@@ -1423,6 +1423,82 @@
+        "category" : ["MUTCAT_LAMIA"],
+        "armor" : [ 
+            { "parts" : "LEG_L", "bash" : 11, "cut" : 24 }, 
+            { "parts" : "LEG_R", "bash" : 11, "cut" : 24 },
+            { "parts" : "FOOT_L", "bash" : 11 , "cut" : 24 },
+            { "parts" : "FOOT_R", "bash" : 11 , "cut" : 24 },
+            { "parts" : "MOUTH", "cut" : 14}],
+        "encumbrance_always" : [
+            [ "LEG_L", 15 ],
+            [ "LEG_R", 15 ],
+            [ "FOOT_L", 15 ],
+            [ "FOOT_R", 15 ]]
+    },{
+        "type" : "mutation",
+        "id" : "LAMIA_SCALES",
+        "name" : "Lamia Scales",
+        "points" : 4,
+        "visibility" : 10,
+        "ugliness" : 1,
+        "description" : "You have grown a smooth, scale-like skin over your legs and mouth. While luxuriously soft, they also give some small resistance to acid",
+        "cancels" : ["SCALES", "FEATHERS", "CHITIN", "PLANTSKIN", "FELINE_FUR", "LUPINE_FUR", "CHITIN_FUR"],
+        "category" : ["MUTCAT_LAMIA"], "//" : "Yes, this is intended to be pre-thresh. You can end up with snake scales regardless if you have a tail or not",
+        "armor" : [
+            { "parts" : "LEG_L", "acid" : 3},
+            { "parts" : "LEG_R", "acid" : 3},
+            { "parts" : "FOOT_R", "acid" : 3},
+            { "parts" : "FOOT_L", "acid" : 3}]
+    },{
+        "type" : "mutation",
+        "id" : "LAMIA_TAIL",
+        "name" : "Snake Tail",
+        "points" : 14,
+        "visibility" : 15,
+        "ugliness" : 6,
+        "description" : "Your legs now terminate in an enormous tail. You can now move greatly faster.",
+        "cancels" : ["SCALES", "FEATHERS", "CHITIN", "PLANTSKIN", "FELINE_FUR", "LUPINE_FUR", "CHITIN_FUR"],
+        "prereqs" : ["LAMIA_TAIL_JEUVENILE"],
+        "threshreq" : ["THRESH_LAMIA"],
+        "category" : ["MUTCAT_LAMIA"],
+        "purifiable" : false,
+        "restricts_gear" : [ "FOOT_L", "FOOT_R", "LEG_L", "LEG_R"],
+        "passive_mods" : {
+            "str_mod" : 1,
+            "dex_mod" : 2 }
+    },{
+        "type" : "mutation",
+        "id" : "LAMIA_TAIL_JEUVENILE",
+        "name" : "Short Snake Tail",
+        "points" : 14,
+        "visibility" : 15,
+        "ugliness" : 6,
+        "description" : "Your legs have fused together to form the beginings of a snake tail. While inherently useless now, you hope it will grow much bigger",
+        "cancels" : ["SCALES", "FEATHERS", "CHITIN", "PLANTSKIN", "FELINE_FUR", "LUPINE_FUR", "CHITIN_FUR", "LEG_TENTACLES", "PADDED_FEET"],
+        "threshreq" : ["THRESH_LAMIA"],
+        "category" : ["MUTCAT_LAMIA"],
+        "changes_to" : ["LAMIA_TAIL"],
+        "purifiable" : true,
+        "restricts_gear" : [ "FOOT_L", "FOOT_R", "LEG_L", "LEG_R"],
+        "passive_mods" : {
+            "dex_mod" : -2},
+        "encumbrance_always" : [
+            [ "LEG_L", 50 ],
+            [ "LEG_R", 50 ],
+            [ "FOOT_L", 50 ],
+            [ "FOOT_R", 50 ] ]
```

Comment:
Here and in every similar example above:
Bracket should be closed on a new line, unless the whole block is on one line.
For example:
```
"one_line" : [ "ONE", "LINE" ],
"many_lines" : [
    "MANY",
    "LINES"
]